.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E226,E302,E41
+max-line-length = 90
+exclude = tests/*
+max-complexity = 10
```

Comment:
It would be helpful if you could provide some info about your choice of this number.

Reply:
Many lines in the files are about 80-90 characters long, so I chose 90.

Reply:
No, this comment was about your max-complexity value.

Reply:
The documentation says complexity that is greater than 10 is too complex...